brambling/tests/functional/test_daily_digest.py

```diff
@@ -0,0 +1,115 @@
+# encoding: utf-8
+from django.core import mail
+from django.test import TestCase
+from mock import patch, MagicMock
+
+from brambling.management.commands.send_daily_emails import Command
+from brambling.models import Person, Transaction
+from brambling.tests.factories import (
+    PersonFactory,
+    OrganizationFactory,
+    EventFactory,
+    TransactionFactory,
+)
+
+
+class DailyDigestCommandTestCase(TestCase):
```

Comment:
Wondering if there should be a test here for the cutoff time aspect of this mailer.